src/command.rs

```diff
@@ -1,9 +1,8 @@
 use std::{collections::BTreeMap, ffi::OsString, io, path::Path, process::Stdio};
 
-use crate::{
-    emit,
-    errors::{CliError, Error},
-};
+use failure::Fallible;
```

Comment:
Why not do:
```suggestion
use failure::{Error, Fallible};
```
and then not need to do `Error` -> `fallible::Error` below?